.github/workflows/build_fork.yml

```diff
@@ -54,7 +54,14 @@ jobs:
         with:
           python-version: 3.8
 
-      - name: lint
+      - name: install elan
```

Comment:
This step is necessary as the "lint style" workflow now needs lake to run `lake exe lint_style`.